### PR TITLE
Rewards: updates unsupported region error text.

### PR DESCRIPTION
**Pull request by mkarolin**
Fixes brave/brave-browser#25092

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [x] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [x] New files have MPL-2.0 license header
- [x] Adequate test coverage exists to prevent regressions
- [x] Major classes, functions and non-trivial code blocks are well-commented
- [x] Changes in component dependencies are properly reflected in `gn`
- [x] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [x] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: components/brave_rewards/resources/ui/components/modalRedirect/style.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const StyledLoader = styled('div')<{}>`
 
 export const StyledError = styled('div')<{}>`
   text-align: center;
+  white-space: pre-wrap;
```
Comment (zenparsing) [inline components/brave_rewards/resources/ui/components/modalRedirect/style.ts:28]: We're using this to preserve newlines from the message string? We might want `pre-line` if we are only interested in the newlines and not other whitespace.

**Review by zenparsing — APPROVED** [review]

File: components/brave_rewards/resources/ui/components/modalRedirect/style.ts (hunk 1 above)
Comment (mkarolin) [reply components/brave_rewards/resources/ui/components/modalRedirect/style.ts:28]: yes, just for the new lines. I'll update. Thanks!

Comment (szilardszaloki) [reply components/brave_rewards/resources/ui/components/modalRedirect/style.ts]: We already have a way of doing this (via `errorText?: string[]`), see `flaggedWalletModal`.

Comment (mkarolin) [reply components/brave_rewards/resources/ui/components/modalRedirect/style.ts]: Ah nice, thanks @szilardszaloki. Updated.

**Review by szilardszaloki — APPROVED** [review]

**Review by szilardszaloki — APPROVED** [review]

**Comment by GeetaSarvadnya** [comment]
Verification PASSED on

```

Brave | 1.45.43 Chromium: 105.0.5195.102 (Official Build) nightly (64-bit)
-- | --
Revision | 4c16f5ffcc2da70ee2600d5db77bed423ac03a5a-refs/branch-heads/5195_55@{#4}
OS | Windows 10 Version 21H2 (Build 19044.1889)

```
- Verified the description from https://github.com/brave/brave-browser/issues/25092#issue-1356645372 
- Ensured that the new text message is displayed for unsupported regions for **Uphold** and **Gemini** custodians
 **_Your {custodianName} account can't be connected to your Brave Rewards profile at this time. Your {custodianName} account is registered in a country that's not currently supported for connecting to Brave Rewards.
See the [current list of supported regions and learn more](https://support.brave.com/hc/en-us/articles/6539887971469) about connecting a custodial account to Brave Rewards_**. 

Uphold | Gemini
-- | --
image

**Pull request merged**